source/codegen/templates/service.h.mako

```diff
@@ -17,6 +17,7 @@ custom_types = common_helpers.get_custom_types(config)
 (input_custom_types, output_custom_types) = common_helpers.get_input_and_output_custom_types(config, functions)
 resource_repository_deps = service_helpers.get_driver_shared_resource_repository_ptr_deps(config, functions)
 resource_handle_types = service_helpers.get_resource_handle_types(config)
+data_moniker_functions = common_helpers.filter_streaming_functions(functions)
```

Comment:
Similar to `service.cpp.mako`, this also needs to generate entry for only those functions which are not marked for no-codegen. This does have equivalent logic to get functions to generate on line 82/98 below. Move it here `functions_to_generate = common_helpers.filter_proto_rpc_functions(functions)` and then pass that to `filter_streaming_functions` API so that you generate only those moniker APIs that need to be generated.

Reply:
updated